documentation/reference.md

```diff
@@ -199,7 +199,7 @@ e.g. to have two related documents with their own title and web address:
 +------------------------+------------------------------+------------------------+----------------------------------+
 |Related Document:0:Title|Related Document:0:Web Address|Related Document:1:Title|Related Document:1:Web Address    |
 +------------------------+------------------------------+------------------------+----------------------------------+
-|A Document              |http://example.com/adocument  |Another Document        |http://example.com/anotherdocument|
+|A Document              |https://example.com/adocument  |Another Document        |https://example.com/anotherdocument|
```

Comment:
It's not required, but it's good practice to add extra `-`s to the table container rows when changing table contents

Reply:
I think I've addressed this in 1763893 and eb2aefc by adding an extra hyphen on each of the table rows in the appropriate columns